src/client/commands/submit.rs

```diff
@@ -169,6 +171,13 @@ pub async fn submit_computation(
         )
     };
 
+    let log = opts.log;
+    if let JobType::Array(ref array) = job_type {
+        if array.id_count() > SUBMIT_ARRAY_LIMIT && log.is_none() {
+            log::warn!("It's recommended to use --log when submitting large arrays");
```

Comment:
* Check also that both stdout, stderr is not disabled
* Make a message more descriptive (and detailed with the actual number of files) "The job will create XXX number of files for stdout/stderr. You may consider using --log option to stream all outputs into one file". where XXX is 2*number of tasks if both stdout/stderr are not disabled)